h2o-py/tests/testdir_misc/pyunit_pivot_medium.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+
+def pivot():
+    df = h2o.create_frame(rows=1000000,
+                          cols=3,
+                          factors=10,
+                          categorical_fraction=1.0/3,
+                          time_fraction=1.0/3,
+                          real_fraction=1.0/3,
+                          real_range=100,
+                          missing_fraction=0.0,
+                          seed=123)
+    dcol = df.moment(year=df["C1"].year(),month=df["C1"].month(),day=df["C1"].day())
```

Comment:
for normal pivot usage, they can use whatever interval of time they want as long as its millis or larger. Here im chopping off the HH:MM:ss part because this is a random frame and without this step the result would be very sparse and not useful to analyze.